content/guides/outliers.md

```diff
@@ -33,15 +33,15 @@ For example, here is a graph of gunicorn requests by host with outlier detection
 
 You can see that one of the series is an outlier: it is handling significantly lower traffic than the others for the time window in question.
 
-To set up an outlier detection graph for your data you add a metric to the graph showing all series in the groups. You apply the outlier detection algorithm by adding `outliers` function on your data. After applying the function, outlier series will be colored with a bold, warm palette, while all other series will be colored with a lightweight, greyscale color palette.
+To set up an outlier detection graph for your data you add a metric to the graph showing all series in the groups. You apply the outlier detection algorithm by adding the `outliers` function on your data. After applying the function, outlier series will be colored with a bold, warm palette, while all other series will be colored with a lightweight, greyscale color palette.
 
 To do so, create a new timeseries graph on your dashboard with your chosen metric. Your screen should look like:
```

Comment:
there are two sentences that talk about setting up an outlier detection graph...or at least I assume there is. The second says 'to do so' so I have to guess what its about. can you clarify this? If i got it right, then which of the two steps do I follow?